source/production/F0.Analyzers/CodeAnalysis/CodeRefactorings/ObjectInitializer.cs

```diff
@@ -212,25 +219,25 @@ private static bool HasDefaultLiteralFeature(Project project)
 			static ISymbol? GetLocalSymbol(ISymbol member, IEnumerable<ILocalSymbol> localSymbols)
 			{
 				return localSymbols
-					.SoleOrDefault(s => s.Name.Equals(member.Name, StringComparison.OrdinalIgnoreCase) && s.Type == GetMemberType(member));
+					.SoleOrDefault(s => s.Name.Equals(member.Name, StringComparison.OrdinalIgnoreCase) && s.Type.Equals(GetMemberType(member), SymbolEqualityComparer.Default));
```

Comment:
Who the heck is this :D ?

Reply:
see https://github.com/Flash0ver/F0.Analyzers/blob/1fe72fd388988d9027dd604dc76dbec648a66053/source/production/F0.Analyzers/Extensions/EnumerableExtensions.Linq.cs#L29-L53